### DISCUSSION TITLE
Performance monitoring not working

### DISCUSSION THREAD

**Discussion by adiig7** [nuxt-community/sentry-module#704] [Q&A] [answered]
<img width="351" alt="image" src="https://github.com/user-attachments/assets/e7089902-207d-47a6-849e-873282f6fc59" />
<br />
<br />
I have setted up the config for performance monitoring, restarted the website on local, but still the performance section is empty.
<br />
<br />
<img width="1080" alt="image" src="https://github.com/user-attachments/assets/58516215-4dc6-4ec8-abe6-78cba41bf392" />

**Comment by rchl** [comment]
What's your full sentry config (in text form)?

I can already see that you are setting some invalid values or properties that are not supported.

Typescript can help to see what options are valid and available for the module. Check at https://sentry.nuxtjs.org/getting-started/setup#types how to make types work.

**Reply by adiig7** [reply]
```
sentry: {
    dsn: "....",
    tracesSampleRate: 1.0,
    browserTracing: true,
    vueOptions: {
      trackComponents: true,
    },
    vueRouterInstrumentationOptions: {
      routeLabel: 'name',
    },
    tracePropagationTargets: ["localhost"],
  }
  ```
  this is my sentry configuration in the `nuxt.config.ts` and have `@nuxtjs/sentry` in my `tsconfig.json` types list too.
  
  what should be correct config for sentry then according to you @rchl?

**Reply by rchl** [reply, marked as answer]
Just set correct `dsn` and `tracing: true`. Remove the rest.

**Reply by adiig7** [reply]
yes working now! thank you!
